starter-code/index.html

```diff
@@ -3,40 +3,95 @@
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
-</body>
-</html>
+  
+  <div class="wrapper">
+    <!-- 
+      <img src="images/landing.jpg" alt="landing">
+      
+    -->
+    
+    <div class="header">
+      <div class="logo-jump">
+        <img src="images/spotify-logo.png" alt="logo" class="logo"/>
+      </div>
+      <div class="nav">
+        <ul>
+          <li>Premium</li>
+          <li>Discover</li>
+          <li>Help</li>
+          <li>Download</li>
+        </ul>
+      </div>
+    </div>
+    
+    <div class="banner">
+      <h1>Music for everyone.</h1>
+      <p>Spotify is now free on mobile, tablet and computer.<br>Listen to the right music, wherever you are.</p>
+    </div>
+    
+    
+    <div class="feature-section">
+      
+      <h2>What’s on Spotify?</h2>
+      
+      <div class="feature">
+        <img src="images/devices-icon.png" alt="devices" class="icon">
+        <div class="name">Millions of Songs</div>
+        <div class="text">There are millions of<br>songs on Spotify</div>
+      </div>
+      
+      <div class="feature">
+        <img src="images/high-quality-icon.png" alt="hq icon" class="icon">
+        <div class="name">HD Music</div>
+        <div class="text">Listen to music as if you<br>were listening live</div>
+      </div>
+      
+      <div class="feature">
+        <img src="images/music-icon.png" alt="music" class="icon">
+        <div class="name">Stream Everywhere</div>
+        <div class="text">Stream music on your<br>smartphone, tablet or<br>computer</div>
+      </div>
+    </div>
+    
+    <div class="preview-section">
+      
+      <div class="col1">
+        <span id="easy">It’s as yeezy as Kanye West.</span> 
+        <div class="preview">
+          <div class="name">Search</div>
+          <div class="desc">Know what you want to listen to?
+            Just search and hit play.</div>
+            
+          </div>
+          <div class="preview">
+            <div class="name">Browse</div> 
+            <div class="desc">Check out the latest charts,
+              brand new releases and great
+              playlists for right now.</div>
+              
+            </div>
+            <div class="preview">
+              <div class="name">Discover</div> 
+              <div class="desc">Enjoy new music every Monday
+                with your own personal playlist.
+                Or sit back and enjoy Radio.</div>
+              </div>
+            </div>
+            
+            <div class="col2">
+              <img src="images/spotify-icon-white.png" alt="spotify icon" id="white-logo"> 
```

Comment:
You can also put the image as a background-image
But it works also like this :)